sdk/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -256,6 +259,11 @@ public CarbonWriterBuilder withLoadOptions(Map<String, String> options) {
         if (escapeChar.length() > 1 && !CarbonLoaderUtil.isValidEscapeSequence(escapeChar)) {
           throw new IllegalArgumentException("ESCAPECHAR cannot be more than one character.");
         }
+      } else if (entry.getKey().equalsIgnoreCase("line_separator")) {
+        String lineSeparator = CarbonUtil.unescapeChar(entry.getValue());
+        if (lineSeparator.isEmpty() || lineSeparator.length() > 2) {
```

Comment:
same as above comments, need to validate whether line separator is validate line separator or not

Reply:
> same as above comments, need to validate whether line separator is validate line separator or not

Univocity parser allow any one or two characters to be the line separator, we can just do the same. User may want use 'a' as line separator.

Reply:
> > same as above comments, need to validate whether line separator is validate line separator or not
> 
> Univocity parser allow any one or two characters to be the line separator, we can just do the same.

Since line separator has no usage in sdk, remove the changes.